src/main/java/com/aerotrack/infrastructure/constructs/RefreshConstruct.java

```diff
@@ -19,12 +21,22 @@
 import java.util.List;
 
 import static com.aerotrack.utils.Constant.*;
```

Comment:
er male totale

Reply:
non l'avevo visto prima

Reply:
giuro che è intellij che me lo mette in automatico

Reply:
devi disattivare così 
https://www.jetbrains.com/help/idea/creating-and-optimizing-imports.html#disable-wildcard-imports